amm/runtime/src/main/scala/ammonite/runtime/ClassLoaders.scala

```diff
@@ -25,22 +25,33 @@ class Frame(val classloader: SpecialClassLoader,
             val pluginClassloader: SpecialClassLoader,
             private[this] var imports0: Imports,
             private[this] var classpath0: Seq[java.io.File]){
+  private var frozen0 = false
+  def frozen = frozen0
+  def freeze(): Unit = {
```

Comment:
Can you add a comment here explaining what the whole point of this freezing thing is? That way the knowledge will remain in the codebase even after further file edits/moves mangles the history and makes the commit message hard to find

Reply:
yep